### ISSUE TITLE
Crashes on Mac

### ISSUE THREAD

**Issue by IAmCheeseman** [samsface/godot-discord-game-sdk#8]
Though it doesn't say Mac support, I might still wanna make Mac versions, just without this feature. I was able to make it just not crash by checking the OS and then not running any functions, but it still throws a few errors, which can make it hard to debug sometimes.

**Comment by samsface** [comment]
Yeah there's no Mac support because discord didn't sign their dylib so modern MacOS refueses to load it. I think there needs to be a safe way to use this in mac projects for sure.
